types/bootstrap-menu/bootstrap-menu-tests.ts

```diff
@@ -0,0 +1,12 @@
+import BootstrapMenu from "bootstrap-menu";
+
+BootstrapMenu("#main", {
```

Comment:
The documentation shows this as a class instead. `var a = new BoostrapMenu(...)`